packages/gestalt/dist/index.d.ts

```diff
@@ -493,6 +493,9 @@ interface DefaultLabelProviderProps {
         };
         SideNavigation: {
           accessibilityDismissButtonLabel: string;
+          accessibilityCollapseButtonLabel: string;
+          accessibilityExpandButtonLabel: string;
+          accessibilityEllipsisLabel: string;
```

Comment:
i think we need to update `packages/gestalt/src/contexts/DefaultLabelProvider.jsdom.test.js` as well

Reply:
and docs here: `docs/examples/defaultlabelprovider/translations.js`

Reply:
Whole discussion here https://pinterest.slack.com/archives/G01GN7CE99S/p1711367607564319